go/testdata/planalyze-vschema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "keyspaces": {
+    "main": {
+      "sharded": true,
+      "vindexes": {
+        "xxhash": {
+          "type": "xxhash"
+        }
+      },
+      "tables": {
```

Comment:
I believe this only works if the keyspace is defined as `main`. We have to make it generic if that is the case.